client/components/mma/delivery/records/ReadOnlyAddressDisplay.tsx

```diff
@@ -58,7 +59,10 @@ export const ReadOnlyAddressDisplay = (props: ReadOnlyAddressDisplayProps) => {
 						${ddCss}
 					`}
 				>
-					<DeliveryAddressDisplay {...props.address} />
+					<DeliveryAddressDisplay
+						data-qm-masking="blocklist"
```

Comment:
Looks like this one got missed when removing the data attribute from components.

Reply:
Removed!